# Definition of ROS service LocAutoStartSavePose for sick localization
# Saves the current pose. If no further pose writing occurs the system will reinitialize itself at this position on restart if LocAutoStart is enabled.
# Example call (ROS1):
# rosservice call LocAutoStartSavePose "{}"
# Example call (ROS2):
# ros2 service call LocAutoStartSavePose sick_lidar_localization/srv/LocAutoStartSavePoseSrv "{}"
# 

# 
# Request (input)
# 

---

# 
# Response (output)
# 

bool success # true: success response received from localization controller, false: service failed (timeout or error status from controller)
